### ISSUE TITLE
Yes, again... Models are invinsible

### ISSUE THREAD

**Issue by AgusUE4** [Godlander/objmc#58]
Hi creators. I already check for the solutions you gave to those who had the same problem as I have. I already tried installing every Minecraft Version (1.18.x ; 1.19.x ; 1.20.x), all of them vanilla, and my custom models are still invinsible. I tried replacing glass blocks and items like potions. But it didnt work. I´ll send all my files so one of you can help me with it, if you can. Also a screenshot from Blender, showing the model, (YES, it has just one Texture) I really want this to work because a have a huge project in mind. 

I´ll wait for help. THANKS!

![imagen_2024-03-01_194736310](https://github.com/Godlander/objmc/assets/66893018/32a4badc-ee6f-4647-a92b-79f0dbfec89d)
![manzino](https://github.com/Godlander/objmc/assets/66893018/474a8bb5-19cf-429f-9f40-d151852fc9fa)
[white_stained_glass.json](https://github.com/Godlander/objmc/files/14466709/white_stained_glass.json)

**Comment by Godlander** [comment]
if the provided sample models don't work, then it is likely a setup or hardware issue

if the provided samples work but your model doesn't work, it is likely user issue. double check your steps again

**Comment by AgusUE4** [comment]
Yes i do understand that. But look at this screenshot. Somehow the model and texture works, but when i place it, becomes invinsible.
![2024-03-04_22 47 57](https://github.com/Godlander/objmc/assets/66893018/9e9d1dfd-e684-47a2-bcfe-b4dd440cfcb5)

**Comment by Godlander** [comment]
are you using sodium?

the readme explains sodium incompatibility. check that you are in vanilla latest release
